ui/js/search.js

```diff
@@ -119,8 +124,9 @@ require(['jquery', 'oae.core', 'underscore', 'jquery.history', 'jquery.switchtab
             // "My" search never includes users, and will be harmful if only "user" is selected from
             // the "Everything" tab. Just always filter it out
             searchParams.resourceTypes = _.without(searchParams.resourceTypes, 'user');
-        } else if (searchParams.scope === '_all' && queryData.tenant) {
-            // If a tenant is specified, we search by the tenant
+        } else if (searchParams.scope === '_all' && (queryData.tenant || isPrivate)) {
+            // If a tenant is specified or the current tenant is private, we search by the tenant
```

Comment:
I don't think it would be since the scope can be changed by changing the tab. Noted on `oae.data.me` though.